src/main/java/com/jjoe64/graphview/Viewport.java

```diff
@@ -830,7 +835,7 @@ public boolean onTouchEvent(MotionEvent event)
                 mIsFirstScrollEvent = true;
             }
         }
-        return b;
+        return false;
```

Comment:
Hi @pavelmakarovswtec - why are we always returning false here?  If this is correct then we  no longer need the variable `b` and it can be removed.